docs/source/upcoming_release_notes/1391-SL2K4_Slit_GUI_Label_Fix.rst

```diff
@@ -0,0 +1,30 @@
+1391 SL2K4 Slit GUI Label Fix
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- N/A
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- Labels no longer clip in control room screens
```

Comment:
We should be specific to which screens we're fixing.  You haven't fixed every label on all the screens, unfortunately 😆